packages/adblocker-webextension/src/index.ts

```diff
@@ -128,19 +139,98 @@ export function updateResponseHeadersWithCSP(
   return { responseHeaders };
 }
 
+const HTML_FILTERABLE_MIME_TYPES = new Set([
```

Comment:
`text/html`?

Reply:
I see `text/xxx` will be included by default below. Let's document this behavior here as well please.

Reply:
Leaving reference [`c71579a` (#14)](https://github.com/seia-soto/adblocker/pull/14/commits/c71579a417157d5166b4efe58a8369aacd7baccb#diff-9545dc27a6748320d2b161f59633758bbfb8482494289e9181fcb5a85021b792R142)